docs/getting_started/execution-modes.rst

```diff
@@ -72,7 +72,10 @@ In this case, users are responsible for declaring which version of ``dbt`` they
 
 Similar to the ``local`` execution mode, Cosmos converts Airflow Connections into a way ``dbt`` understands them by creating a ``dbt`` profile file (``profiles.yml``).
 
-A drawback with this approach is that it is slower than ``local`` because it creates a new Python virtual environment for each Cosmos dbt task run.
+Some drawbacks of this approach:
+
+- It is slower than ``local`` because it creates a new Python virtual environment for each Cosmos dbt task run.
+- The ``dbt_ls`` load method is not supported with this execution mode. See `Parsing Methods <parsing-methods.html>`_ for more information.
```

Comment:
The ``dbt_ls`` load method is indifferent to the execution mode - its only expectation is to have a dbt binary accessible from the instance running the Airflow scheduler. Therefore, I don't think it's accurate.

```suggestion
- If dbt is unavailable in the Airflow scheduler, the default ``LoadMode.DBT_LS`` will not work. In this scenario, users must use a `parsing method <parsing-methods.html>`_  that does not rely on dbt, such as ``LoadMode.MANIFEST``.
```